docs/src/reference-guide/components/indexers/index.md

```diff
@@ -0,0 +1,8 @@
+# Indexers
+
+Indices are the one of the core building blocks of the Fuel indexer service. Indices allow indexer operators and dApp authors to index specific blockchain events into a database.
```

Comment:
In `docs/src/the-fuel-indexer.md` we refer to Indexer plural as Indexers, here we say Indices. Let's stay consistent, I don't mind which one we choose, personally I prefer Indexers. Also, The title of this section is # Indexers then we immediately say `Indices`, a bit confusing.

Reply:
Oh whoops, that was a find-and-replace gone wrong. Will fix.